src/actions/__test__/rateArticleActions.test.js

```diff
@@ -0,0 +1,51 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import * as types from "../actionTypes";
+import * as actionCreators from "../rateArticleActions";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+// test action creators
+describe("get average rates action", () => {
+  it("should return a get action", () => {
+    const store = mockStore({});
```

Comment:
You can have the constants set before the test suite to avoid repetition.

Reply:
True. Let me work on it